models/User.js

```diff
@@ -0,0 +1,30 @@
+import mongoose from "mongoose";
+
+/*
+ * To be honest, with our OAuth provider, we don't really need a User model.
+ * However, we may want to add some additional fields to our User that isn't 
+ * provided by our OAuth provider for Beavs.ai, so we'll create a User model for now.
+*/
+const UserSchema = new mongoose.Schema({
+      /* 
+       * required: true means that the field is mandatory to "create" a new User
+       * in mongodb, the primary key "_id" is automatically generated
+       * mongoose types: https://mongoosejs.com/docs/schematypes.html#what-is-a-schematype
+      */
+      name: {
+            type: String,
+            required: true,
+      },
+      major: {
+            type: String,
+            required: true,
+      },
+      standing: {
+            type: String,
```

Comment:
Is string the best type for this?

Reply:
Yep, mongoose supports an [enum](https://mongoosejs.com/docs/schematypes.html#what-is-a-schematype:~:text=given%20regular%20expression-,enum,-%3A%20Array%2C%20creates%20a) validator array. We could however make it more obvious.